recipes/ccache/all/conanfile.py

```diff
@@ -26,42 +31,44 @@ class CcacheConan(ConanFile):
         "redis_storage_backend": True,
     }
 
-    generators = "cmake", "cmake_find_package_multi"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _is_msvc(self):
-        return str(self.settings.compiler) in ["Visual Studio", "msvc"]
-
     @property
     def _min_cppstd(self):
-        return "17" if self._is_msvc else "14"
+        if Version(self.version) > "4.7":
+            return "17"
+        else:
+            return "17" if is_msvc(self) else "14"
 
     @property
     def _compilers_minimum_version(self):
-        return {
-            "gcc": "6",
-            "clang": "6",
-            "apple-clang": "10",
-            "Visual Studio": "15.7" if tools.Version(self.version) < "4.6" else "16.2",
-        }
+        if Version(self.version) > "4.7":
+            return {
+                "gcc": "8",
+                "clang": "9",
+                "apple-clang": "11",
+                "Visual Studio": "16.2",
+            }
+        else:
+            return {
+                "gcc": "6",
+                "clang": "6",
+                "apple-clang": "10",
+                "Visual Studio": "15.7" if Version(self.version) < "4.6" else "16.2",
```

Comment:
There's a new msvc compiler for 2.0, you can use this table to convert to the new versions